### PR TITLE
Add keycard policy

### PR DESCRIPTION
**Pull request by kpengboy**
Can it be officially promulgated?

### CODE REVIEW

File: ocfweb/docs/docs/staff/policies/keycard.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+<!-- This policy was promulgated by the GMs on 8/18/2017. -->
+
+
+## Keycard Access
+
+Keycard access is given much like root, but requires approval from both GMs and
+SMs.
+
+To be eligible for Keycard access, you must fulfill the following criteria:
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/policies/keycard.md]: `Keycard` -> `keycard`

File: ocfweb/docs/docs/staff/policies/keycard.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+[[!meta title="Keycard policy"]]
+
+<!-- This policy was promulgated by the GMs on 8/18/2017. -->
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/policies/keycard.md]: iso8601 pls

File: ocfweb/docs/docs/staff/policies/keycard.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+To be eligible for Keycard access, you must fulfill the following criteria:
+
+ - A Site Manager/General Manager must be able to recognize you in person
+ - You must have been a staffer for at least a year already, but this may be
+   waived at the discretion of the GM/SM
+ - You must demonstrate a concrete, actual need for keycard privileges and
+   concrete examples of helpful activity in the OCF
+
+Keycard Access, once given, can be revoked at the discretion of SMs or GMs.
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/policies/keycard.md]: `Access` -> `access`? why are these things being arbitrarily capitalized in only some places? lol

File: ocfweb/docs/docs/staff/policies/keycard.md (hunk 2 above)
Comment (kpengboy) [reply ocfweb/docs/docs/staff/policies/keycard.md]: Actually I'll spell out the date instead to harmonize it with the other policies.

File: ocfweb/docs/docs/staff/policies/keycard.md (hunk 1 above)
Comment (kpengboy) [reply ocfweb/docs/docs/staff/policies/keycard.md]: Done

File: ocfweb/docs/docs/staff/policies/keycard.md (hunk 3 above)
Comment (kpengboy) [reply ocfweb/docs/docs/staff/policies/keycard.md]: Ask Sahil...

**Pull request merged**